README.md

```diff
@@ -13,7 +13,7 @@
 [![Build Docker Image](https://github.com/drifting-in-space/plane/actions/workflows/plane2-build-image.yml/badge.svg)](https://github.com/drifting-in-space/plane/actions/workflows/plane2-build-image.yml)
 [![Tests](https://github.com/drifting-in-space/plane/actions/workflows/plane2-tests.yml/badge.svg)](https://github.com/drifting-in-space/plane/actions/workflows/plane2-tests.yml)
 
-Plane is infrastructure software for **running stateful WebSocket backends at scale**. Plane is heavily inspired by [Figma’s mulitplayer infrastructure](https://www.figma.com/blog/rust-in-production-at-figma/), which dynamically spawns a process for each active document.
+Plane is a distributed system for **running stateful WebSocket backends at scale**. Plane is heavily inspired by [Figma’s mulitplayer infrastructure](https://www.figma.com/blog/rust-in-production-at-figma/), which dynamically spawns a process for each active document.
```

Comment:
what does it mean for plane to be a distributed system - is there a specific type of system it is?

Reply:
Here's a good definition:

> A distributed system is a system whose components are located on different networked computers, which communicate and coordinate their actions by passing messages to one another from any system. The components interact with one another in order to achieve a common goal.